react/features/recording/components/LiveStream/StartLiveStreamDialog.web.js

```diff
@@ -341,6 +338,42 @@ class StartLiveStreamDialog extends Component {
             });
     }
 
+    /**
+     * Takes in a list of broadcasts from the YouTube API, removes dupes,
+     * removes broadcasts that cannot get a stream key, and parses the
+     * broacasts into flat objects.
```

Comment:
`broacasts` -> `broadcasts`

Reply:
I've gone ahead and enabled spellcheck on my editor by default. Let's see how much that helps me.